bodhi/server/consumers/signed.py

```diff
@@ -101,3 +107,13 @@ def __call__(self, message: fedora_messaging.api.Message):
             log.info("Build has been signed, marking")
             build.signed = True
             log.info("Build %s has been marked as signed" % build_nvr)
+
+            # If every build in update is signed change status to testing
+            if build.update.from_tag and build.update.signed():
+                log.info("Every build in update is signed, set status to testing")
+                from sqlalchemy import func
```

Comment:
Let's put this import with the rest of the imports at the top of the file :-)

Reply:
Moved to the import section :-)